arch/arm64/src/common/arm64_arch.h

```diff
@@ -299,6 +299,9 @@ struct regs_context
   uint64_t  spsr;
   uint64_t  sp_el0;
   uint64_t  exe_depth;
+#ifdef CONFIG_ARCH_FPU
```

Comment:
Is the content of this patch needed by SMP too ?

Reply:
The removable of FPU trap reason:
1. The FPU trap logic is not stable in some SMP situation (HERE haven't the detail analysis)
2. Make the logic Complex and different to read.
3. The most important reason: it has almost no benefit for the performance, because the compiler will optimize the code with float/double instructions when use the option -O3 (always be). Even the printf/syslog will use use the FPU, so the FPU is used too often and almost each context switch.
Or, you must use -mgeneral-regs-only to limit the FPU registers usage, that will harmful for the speed.
5. How about use the FPU registers in the IRQ/SVC ISR, maybe there are hidden bugs ?